web/src/js/__tests__/components/FlowTable/FlowColumnsSpec.js

```diff
@@ -1,101 +1,101 @@
 import React from 'react'
 import renderer from 'react-test-renderer'
 import * as Columns from '../../../components/FlowTable/FlowColumns'
-import { TFlow } from '../../ducks/tutils'
+import _tflow from '../../ducks/_tflow'
```

Comment:
Do you want to have `tflow` as a "public" module or do we want to integrate it in `tutils`? I was thinking of the following in tutils:
```javascript
export {default as TFlow} from './_tflow'
```
Both really works, go with what you prefer. 😃